astro/src/content/docs/apis/_import-users-request-body.mdx

```diff
@@ -141,12 +141,16 @@ You must provide either the **email** or the **username** field for each User. T
   </Aside>
 </APIField>
 
-<APIField name="users[x].password" type="String" required>
+<APIField name="users[x].password" type="String" optional>
   The User's password. If <InlineField>encryptionScheme</InlineField> is provided this value is the existing password hash, if <InlineField>encryptionScheme</InlineField> is not provided this value is assumed to be plain text and it will be hashed during import.
 
   The value of <InlineField>password</InlineField> should contain only the password hash, or, if the <InlineField>encryptionScheme</InlineField> is omitted, the plain text password.  Unless you are using a custom plugin that does not utilize base64, <InlineField>password</InlineField> should be base64 encoded.  It is recommended to use a base64 encoded string, but if you are importing hashes from a legacy system that uses base16, base32, or another encoding and are writing a custom plugin, import the hash in the same format the plugin produces.
 
   Please see the [Password Hashes](#password-hashes) section for more information about importing particular password hashes.
+
```

Comment:
Was not sure how to message this. The field has been around for a while but becomes optional in 1.99.9

Reply:
Can we say something like:

Prior to version 1.99.9 this field is required. Beginning with version 1.99.9 this field is only required when the tenant you are importing users into requires passwords.

Reply:
Yes, but we forgot to link import validation with that. Created ENG-2591.

Reply:
Pushed